File: app/src/main/java/com/bubelov/coins/domain/ExchangeRate.java
package com.bubelov.coins.domain;

/**
 * @author Igor Bubelov
 */

public class ExchangeRate {
    public String source;

    public String baseCurrencyCode;

    public String targetCurrencyCode;

    public double rate;

    public ExchangeRate(String source, String baseCurrencyCode, String targetCurrencyCode, double rate) {
        this.source = source;
        this.baseCurrencyCode = baseCurrencyCode;
        this.targetCurrencyCode = targetCurrencyCode;
        this.rate = rate;
    }
}

File: app/src/main/java/com/bubelov/coins/model/ExchangeRate.kt
package com.bubelov.coins.model

/**
 * @author Igor Bubelov
 */

data class ExchangeRate(
        val id: Long,
        val source: String,
        val baseCurrencyCode: String,
        val targetCurrencyCode: String,
        val rate: Double,
        val date: Long
)